modules/solid_mechanics/doc/content/source/bcs/DirectDirichletBC.md

```diff
@@ -0,0 +1,21 @@
+# DirectDirichletBC
+
+!syntax description /BCs/DirectDirichletBC
+
+## Overview
+
+This applies a Dirichlet BC meant to be used in conjunction with [DirectCentralDifference](source/timeintegrators/DirectCentralDifference.md).
+
+BC's are applied by calculating the residual force needed,$\mathbf{F}_g$, to enforce BC's during a central difference solution update.
+
+At each boundary node:
+
+\begin{equation}
+    F_g = \frac{u_g-u_n}{\Delta t^2} - \frac{v_{n-\frac{1}{2}}}{\Delta t}
+\end{equation}
```

Comment:
Explain the meaning of the variables.

Reply:
Provide an equation for Delta t for changing step size.

Reply:
[a2a74d6](https://github.com/idaholab/moose/pull/28175/commits/a2a74d6e380e6d7fbbd23093bb3390cfa0741a73)